src/pages/crew/allMembers.jsx

```diff
@@ -0,0 +1,98 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Button } from "../../components/button"
+import { useHistory } from "react-router-dom";
+import { SimpleUserCard } from "../../components/simpleUserCard";
+
+import styled from "styled-components";
+import { Theme } from "../../theme";
+
+const S = {
+    CrewList: styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    `,
+    CrewMember: styled.div`
+        border: 1px solid;
+        padding: 1em;
+        width: 20em;
+    `,
+    CrewAvatar: styled.img`
+        max-width: 9em;
+    `,
+}
+
+export const CrewMemberList= () => {
+    const [crews, setCrews] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [visibleUUID, setVisibleUUID] = useState(false);
+
+    let history = useHistory();
+
+    useEffect(async () => {
+        const crews = await Promise.all((await Crew.getCrews())
+            .filter(crew => crew.active)
+            .map(async (crew) => {
+                return await Crew.getCrew(crew.uuid);
+            })
+        )
+        setCrews(crews);
+        setLoading(false);
+    }, []);
+
+    if(loading) {
+        return (
+            <PageLoading />
+        )
+    }
+
+    return (
+        <>
+            <DashboardHeader border>
+                <DashboardTitle>
+                    Alle crewmedlemmer
+                </DashboardTitle>
+                <DashboardSubtitle>
+                    Viser {crews.length} aktive crew
+                </DashboardSubtitle>
+            </DashboardHeader>
+            <DashboardContent>
+                <InnerContainer>
+                    {
+                        crews.map((crew) => {
+                            const crewMembersMap = new Map();
+                            crew.positions.forEach((position) => {
+                                position.position_mappings.map(mapping => mapping.user).forEach((user) => {
+                                    if(!crewMembersMap.has(user.uuid)) {
+                                        crewMembersMap.set(user.uuid, user)
+                                    }
+                                })
+                            })
+
+                            const crewMembers = Array.from(crewMembersMap.values());
+                            console.log(crewMembers)
+
+                            return (
+                                <>
+                                    <InnerContainerTitle nopadding>{crew.name}</InnerContainerTitle>
+                                    <p><i>{crewMembers.length} medlemmer</i></p>
+                                    <InnerContainerRow>
+                                        {
+                                            crewMembers.map(member => (<SimpleUserCard user={member} key={member.uuid} />))
+                                        }
+                                    </InnerContainerRow>
```

Comment:
![image](https://github.com/user-attachments/assets/2294fa24-5bed-40d0-be28-7dfc5ce6abd3)
The margin between the crew name and "Se crew" button is non-existing, should have `<br />` or margin-top/margin-bottom property.
I suggest putting each crew in their own `<div>`, with title, members and a button at bottom to view their crew. And wrap these in a "head" `<div>` with `display: flex; gap: 2em`

I also believe the button *can* be associated with the crew name below the button.